graphql/resolvers.go

```diff
@@ -1254,6 +1254,16 @@ func (r *queryResolver) DistroTaskQueue(ctx context.Context, distroID string) ([
 	return taskQueue, nil
 }
 
+func (r *taskQueueItemResolver) ExpectedDuration(ctx context.Context, obj *restModel.APITaskQueueItem) (string, error) {
```

Comment:
Wouldn't it be cleaner to return this as a duration type and let the client format it?

Reply:
seems fine to me this way. this is how it's done for the patch metadata.

Reply:
Arguably the better option would have been to have this be a duration and format on the client, since we already have a couple of formatting functions there. Formatting on the server also makes it harder to do localization

Reply:
PatchTime relies on a user's timezone setting in the db to format the utc date string which is less ideal compared to the browser which already understands which timezone it's operating in without additional data. 

We are currently formatting durations on the client (https://github.com/evergreen-ci/spruce/blob/master/src/utils/string/index.ts#L46) and server (https://github.com/evergreen-ci/evergreen/blob/master/graphql/util.go#L573) and we should definitely stick to just one moving forward. 

Returning milliseconds from the server and formatting duration on the client seems better because we can assure that we would only ever need to make code changes in Spruce if we ever need to adjust the time/duration display string (instead of having to update both GQL integration test & cypress tests)

Reply:
noted! will change